src/platform/packages/shared/kbn-esql-ast/src/definitions/utils/autocomplete/functions.ts

```diff
@@ -460,18 +450,18 @@ async function getListArgsSuggestions(
       }
     }
 
-    const anyUserDefinedColumns = collectUserDefinedColumns(commands, fieldsMap, innerText);
-    // extract the current node from the userDefinedColumns inferred
-    anyUserDefinedColumns.forEach((values, key) => {
-      if (values.some((v) => v.location === node.location)) {
-        anyUserDefinedColumns.delete(key);
-      }
-    });
+    // FIXME
```

Comment:
This wants to get fixed 😄 Is a leftover?

Reply:
Gone in https://github.com/elastic/kibana/pull/233221/commits/c7a8428adf8f53e422a687789acdebd60b8a6ec7